charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.39.3
+version: 0.39.4
```

Comment:
@povilasv We normally bump the minor version when we do the collector image bump since the collector itself does a minor version bump.